GCTC-NTGC/gc-digital-talent
# Builds the new browse IT jobs page

# 🧪 Requirements
- Translations still required
- Some components should be migrated to the common component library (marked)
- Already responsive and dark-mode enabled

# ✅ Related issues
- Addresses #4272 

# 🧪 Testing

1. Build talent search `npm run production --workspace=talentsearch`
2. Navigate to `/browse/pools`
3. Confirm it matches the [mockup](https://www.figma.com/proto/NhEMp9tiBNi5VwuhmRr4JW/Browse-jobs%2C-Support-and-FAQs?page-id=0%3A1&node-id=116%3A2175&viewport=-3163%2C-684%2C0.25&scaling=min-zoom)
4. Confirm the links work
5. Confirm it gracefully changes as window resizes
